src/backends/backend.js

```diff
@@ -26,7 +26,24 @@ class LocalStorageAuthStore {
 
 const slugFormatter = (template = "{{slug}}", entryData) => {
   const date = new Date();
-  const identifier = entryData.get("title", entryData.get("path"));
+
+  const getIdentifier = (entryData) => {
```

Comment:
Agreed. Commit e51071a implements this behavior. 

I think the original calls in L56 to `.trim` are to trim the value corresponding to the field so that it doesn't mess up the slug, not the field itself (the key).